can/message.py

```diff
@@ -279,8 +279,21 @@ def _check(self):
                     "error state indicator is only allowed for CAN FD frames"
                 )
 
+    def __eq__(self, other: Any) -> bool:
+        """ Compares a given message with this one.
+        The result is the as same as calling ``equals`` without timestamp and channel check.
+
+        :param other: the message to compare with
+
+        :return: True if the given message equals this one
+        """
+        return self.equals(other, timestamp_delta=None, check_channel=False)
+
     def equals(
-        self, other: "Message", timestamp_delta: Optional[Union[float, int]] = 1.0e-6
+        self,
+        other: Any,
+        timestamp_delta: Optional[Union[float, int]] = 1.0e-6,
+        check_channel: bool = False,
```

Comment:
We should probably document backwards incompatible changes somewhere.

Reply:
See above and with check_channel = True this is full backwards compatible.